src/clj_kondo/impl/sarif.clj

```diff
@@ -1,36 +1,48 @@
 (ns clj-kondo.impl.sarif
-  (:require [clj-kondo.impl.config :as config]))
+  (:require
+   [clj-kondo.impl.config :as config]
+   [clj-kondo.impl.version :as version]))
 
 ;; https://github.com/microsoft/sarif-tutorials/blob/main/docs/1-Introduction.md#simple-example
 
 (set! *warn-on-reflection* true)
 
-(defn finding->sarif [rules files finding]
-  {:level (:level finding)
+(defn- linter-help-uri [linter]
+  (format "https://github.com/clj-kondo/clj-kondo/blob/master/doc/linters.md#%s" (name linter)))
+
+(defn- sarif-level [level]
+  (case level
+    :error "error"
+    :warning "warning"
+    :off "none"))
```

Comment:
clj-kondo also supports :info level, this function will crash currently if that level is used somewhere

Reply:
Add :info level